src/CommunityToolkit.Maui/Controls/IAvatarElement.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.ComponentModel;
+
+namespace CommunityToolkit.Maui.Controls;
+
+/// <summary>Avatar content view element interface.</summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
+public interface IAvatarElement
```

Comment:
- Please move this file to the `CommunityToolkit.Maui.Core` project into its `Interfaces` folder
- Please change the namespace to `CommunityToolkit.Maui.Core`
- Please rename this to  `IAvatarView`
- Please have `IAvatarView` inherit from `IContentView, IBorder, IFont, IText, Microsoft.Maui.IImage`
```suggestion
public interface IAvatarView : IContentView, IBorder, IFont, IText, IImageSource, Microsoft.Maui.IImageSourcePart
```
  - Please then remove duplicate properties like `string Text { get; }` which is inherited from `IText`

Reply:
Fully understand and will move as indicated, change the namespace, rename the interface and have it inherit as appropriate.